code/src/LeafletProvider/DrawingTools/DrawingTools.ts

```diff
@@ -0,0 +1,237 @@
+/// <reference path="../../OSFramework/DrawingTools/AbstractDrawingTools.ts" />
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace LeafletProvider.DrawingTools {
+    class ToolsList {
+        //TODO - Create/optimize types for all the tools by using a global.d.ts or ussi
+        //     - In that moment each tool should be boolean | NEW_TYPE because the empty state is false.
+        // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+        public circle: any;
+        public circleMarker: boolean;
+        // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+        public marker: any;
+        // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+        public polygon: any;
+        // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+        public polyline: any;
+        // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+        public rectangle: any;
+
+        constructor() {
+            this.circleMarker = false; // this tool isn't provided by our experience
+            this.circle = false;
+            this.marker = false;
+            this.polygon = false;
+            this.polyline = false;
+            this.rectangle = false;
```

Comment:
Can you add a comment here explaining why do we have a start value = false?
(Refer where this values can be updated and why they have the def value = false)